manager-bundle/skeleton/config/security.yaml

```diff
@@ -33,6 +33,7 @@ security:
             request_matcher: contao.routing.frontend_matcher
             provider: contao.security.frontend_user_provider
             user_checker: contao.security.user_checker
+            access_denied_handler: contao.security.access_denied_handler
```

Comment:
You have to also update the `README.md` in the `core-bundle` for regular installations.

Reply:
And the `core-bundle/tests/Functional/app/config/security.yaml` file.

Reply:
fixed in 9bb19bdf3b732c6a6938d11cacdabf033bc1929f